specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json

```diff
@@ -77,7 +77,10 @@
             "in": "path",
             "description": "The administrator name.",
             "required": true,
-            "type": "string"
+            "type": "string",
+			"enum": [
+				"ActiveDirectory"
```

Comment:
Does the change from string to enum result in a breaking change for some SDKs?

Reply:
Currently, there is a difference in number of parameters for create/modify/delete methods in .NET and Python clients between MI and SQL DB. So, this change is made to aligned MI clients with SQL DB (they need to be auto-generated again based on updated json).
Jared has noticed in PR https://github.com/Azure/azure-cli/pull/10446 that there is parameter AdministratorName in method create_or_update which always takes the same value and that this parameter doesn't exist in clients for sql db.
I am not sure if it is a breaking change.

Reply:
Yes, it is a breaking change. However this is a new API and therefore right now is the time that break will have minimal impact. Also this change brings managed instance admin API into sync with server admin API.